### ISSUE TITLE
Write unit tests

### ISSUE THREAD

**Issue by jamesgeorge007** [madlabsinc/mevn-cli#138]
- [ ] `mevn init`
- [ ] `mevn codesplit`
- [ ] `mevn generate`
- [ ] `mevn add`
- [ ] `mevn serve`
- [ ] `mevn dockerize`
- [ ] `mevn deploy`

**Comment by anooppoommen** [comment]
@jamesgeorge007 Hey is this up for grabs

**Comment by jamesgeorge007** [comment]
@anooppoommen yeah, it is :+1: Let me know the command that you're willing to work on.

**Comment by anooppoommen** [comment]
@jamesgeorge007 I'll work on the mevn init function for now

**Comment by cherryWood55** [comment]
I want to work on this issue.

**Comment by jamesgeorge007** [comment]
@cherryWood55 you may take up a command other than `mevn init`

**Comment by otaviopace** [comment]
Where is the `mevn init` unit test?

**Comment by jamesgeorge007** [comment]
@otaviopace as of now we are missing out on unit tests (testing out the workflow in isolation). Feel free to pick up a command that you wish to work upon.

**Comment by ThakurKarthik** [comment]
@jamesgeorge007 i am taking ```mev info```.Is it okay?

**Comment by meroware** [comment]
@jamesgeorge007 I shall take `generate` if no one else has `https://github.com/madlabsinc/mevn-cli/blob/develop/src/commands/basic/generate.js`. But I did notice that there is no jest setup for unit tests only e2e. I can include the setup in my PR as well, but wondering what's the plan on that part.

**Comment by jamesgeorge007** [comment]
@meroware thanks for showing interest, we don't have any unit tests at the moment. Feel free to shoot a PR.
